website/content/en/docs/contribution-guidelines/issue-lifecycle.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Issue Lifecycle
+weight: 30
+---
+
+The Operator SDK project tracks bugs, feature requests, and questions
+with [GitHub
+issues](https://github.com/operator-framework/operator-sdk/issues).
+
+## Triage
+
+Each week, there is a [community
+meeting](https://github.com/operator-framework/community/#operator-sdk-community-issue-triagegrooming)
+to triage issues. Each issue that has been filed since the previous
+meeting is discussed, GitHub labels are applied, and the issue is added
+to a Milestone. Additionally, anyone can request that a previously
```

Comment:
would it be helpful to link to `Milestone`?  https://github.com/operator-framework/operator-sdk/milestones